src/main/java/teammates/ui/pagedata/InstructorFeedbackResponseCommentAjaxPageData.java

```diff
@@ -41,6 +41,11 @@ public FeedbackResponseCommentRow getComment() {
                                                getResponseVisibilities(), instructorEmailNameTable, sessionTimeZone);
         frc.enableEditDelete();
 
+        boolean isVisibilityIconShown = !frc.getShowCommentToString().isEmpty();
```

Comment:
Add a comment
```
// only show icon if the comment is visible to at least someone
```

On a separate note, shall we just remove the constrain for all `results.feedbackSession.isPublished()`?

Reply:
Yes, Should I do that in this PR or open a new one for it?